### ISSUE TITLE
Unable to upload new assets

### ISSUE THREAD

**Issue by Purzelkater** [Kruptein/PlanarAlly#103]
If I try to upload new assets (images), the console shows some errors. I have tried to upload on the assets root directory and a subdirectory and with drag&drop and file dialog.

```
message async handler error
Traceback (most recent call last):
  File "D:\Python\Python37\lib\site-packages\engineio\asyncio_server.py", line 278, in _trigger_event
    ret = await self.handlers[event](*args)
  File "D:\Python\Python37\lib\site-packages\socketio\asyncio_server.py", line 353, in _handle_eio_message
    pkt.data)
  File "D:\Python\Python37\lib\site-packages\socketio\asyncio_server.py", line 294, in _handle_event
    await self._handle_event_internal(self, sid, data, namespace, id)
  File "D:\Python\Python37\lib\site-packages\socketio\asyncio_server.py", line 297, in _handle_event_internal
    r = await server._trigger_event(data[0], namespace, sid, *data[1:])
  File "D:\Python\Python37\lib\site-packages\socketio\asyncio_server.py", line 325, in _trigger_event
    ret = await self.handlers[namespace][event](*args)
  File "D:\Python\PlanarAlly\PlanarAlly\auth.py", line 43, in wrapped
    return await fn(*args, **kwargs)
  File "D:\Python\PlanarAlly\PlanarAlly\api\asset_manager.py", line 126, in assetmgmt_upload
    folder = functools.reduce(dict.get, file_data["directory"], user.asset_info)
AttributeError: 'User' object has no attribute 'asset_info'
```

**Comment by Kruptein** [comment]
Thanks for the report, I'll fix it asap

**Comment by Kruptein** [comment]
Fixed in 0.11.4

**Comment by Purzelkater** [comment]
Thank you for the fast fix. Upload with file dialog was no problem now, but still error with drag&drop on assets manager (Chrome browser):

```
0/26
2/26
3/26
1/26
4/26
5/26
6/26
7/26
9/26
10/26
12/26
11/26
14/26
13/26
15/26
16/26
8/26
17/26
19/26
20/26
18/26
21/26
22/26
24/26
25/26
23/26
message async handler error
Traceback (most recent call last):
  File "D:\Python\Python37\lib\site-packages\engineio\asyncio_server.py", line 278, in _trigger_event
    ret = await self.handlers[event](*args)
  File "D:\Python\Python37\lib\site-packages\socketio\asyncio_server.py", line 353, in _handle_eio_message
    pkt.data)
  File "D:\Python\Python37\lib\site-packages\socketio\asyncio_server.py", line 294, in _handle_event
    await self._handle_event_internal(self, sid, data, namespace, id)
  File "D:\Python\Python37\lib\site-packages\socketio\asyncio_server.py", line 297, in _handle_event_internal
    r = await server._trigger_event(data[0], namespace, sid, *data[1:])
  File "D:\Python\Python37\lib\site-packages\socketio\asyncio_server.py", line 325, in _trigger_event
    ret = await self.handlers[namespace][event](*args)
  File "D:\Python\PlanarAlly-0.11.4\PlanarAlly\auth.py", line 43, in wrapped
    return await fn(*args, **kwargs)
  File "D:\Python\PlanarAlly-0.11.4\PlanarAlly\api\asset_manager.py", line 131, in assetmgmt_upload
    owner=user, parent=file_data["directory"])
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 5531, in create
    inst.save(force_insert=True)
  File "D:\Python\Python37\lib\site-packages\playhouse\signals.py", line 71, in save
    ret = super(Model, self).save(*args, **kwargs)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 5678, in save
    pk_from_cursor = self.insert(**field_dict).execute()
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 1604, in inner
    return method(self, database, *args, **kwargs)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 1675, in execute
    return self._execute(database)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 2331, in _execute
    return super(Insert, self)._execute(database)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 2097, in _execute
    cursor = database.execute(self)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 2695, in execute
    sql, params = ctx.sql(query).query()
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 558, in sql
    return obj.__sql__(self)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 2306, in __sql__
    self._simple_insert(ctx)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 2194, in _simple_insert
    return self._generate_insert((self._insert,), ctx)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 2278, in _generate_insert
    return ctx.sql(CommaNodeList(all_values))
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 558, in sql
    return obj.__sql__(self)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 1510, in __sql__
    ctx.sql(self.nodes[n_nodes - 1])
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 558, in sql
    return obj.__sql__(self)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 1508, in __sql__
    ctx.sql(self.nodes[i])
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 558, in sql
    return obj.__sql__(self)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 1217, in __sql__
    return ctx.value(self.value, self.converter)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 570, in value
    value = converter(value)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 4492, in db_value
    return self.rel_field.db_value(value)
  File "D:\Python\Python37\lib\site-packages\peewee.py", line 3945, in db_value
    return value if value is None else self.adapt(value)
ValueError: invalid literal for int() with base 10: '.'
```

**Comment by Kruptein** [comment]
Should be fixed in 0.11.5 :D
